cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/AwsClusterSettingsStep.tsx

```diff
@@ -109,12 +83,14 @@ function AwsClusterSettingsStep(props: Partial<StepProps>) {
     if (awsState[STORE_SECTION_FORM][AWS_FIELDS.OS_IMAGE]) {
         setValue(AWS_FIELDS.OS_IMAGE, awsState[STORE_SECTION_FORM][AWS_FIELDS.OS_IMAGE].name);
     }
+
+    if (!initialSelectedInstanceTypeId) {
+        initialSelectedInstanceTypeId = nodeInstanceTypes[0].id;
+        setValue(AWS_FIELDS.NODE_PROFILE, initialSelectedInstanceTypeId);
```

Comment:
It feels like the instance type (example: 't3a.large') should be saved in AWS_FIELDS.NODE_TYPE and the Node Profile value should be saved in AWS_FIELDS.NODE_PROFILE. Maybe I am misunderstanding something, so apologies if I am.

Reply:
Yes sir. That is right

Reply:
Okay then I think we should change the variable name to 'initialSelectedNodeProfileId'

Reply:
Yes sir. For this PR, I will rename the variable. 
By the way this 'initialSelectedInstanceTypeId' is used in both Azure and Vsphere currently. 
I will correct them when adding the 'initNodeProfile' for Azure and Vsphere in the future.
Just want this PR focused in AWS